src/EditorFeatures/VisualBasic/LineCommit/CommitCommandHandler.vb

```diff
@@ -216,43 +220,33 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.LineCommit
         End Function
 
         Public Sub ExecuteCommand(args As PasteCommandArgs, nextHandler As Action, context As CommandExecutionContext) Implements IChainedCommandHandler(Of PasteCommandArgs).ExecuteCommand
-            Using context.OperationContext.AddScope(allowCancellation:=True, VBEditorResources.Formatting_pasted_text)
-                CommitOnPaste(args, nextHandler, context.OperationContext.UserCancellationToken)
-            End Using
-        End Sub
+            Dim oldVersion = args.SubjectBuffer.CurrentSnapshot.Version
 
-        Private Sub CommitOnPaste(args As PasteCommandArgs, nextHandler As Action, cancellationToken As CancellationToken)
-            Using transaction = _textUndoHistoryRegistry.GetHistory(args.TextView.TextBuffer).CreateTransaction(VBEditorResources.Paste)
```

Comment:
this is a pretty big change for VB muscle memory.  Previously, the paste+commit was always in one transaction for VB.  Now it will be either one or two edits ('one' if commit did nothing, 'two' if it did).